### PR TITLE
Setup github actions

### LINKED ISSUE #1: Action required: Greenkeeper could not be activated 🚨
🚨 You need to enable Continuous Integration on Greenkeeper branches of this repository. 🚨

To enable Greenkeeper, you need to make sure that a [commit status](https://help.github.com/articles/about-statuses/) is reported on all branches. This is required by Greenkeeper because it uses your CI build statuses to figure out when to notify you about breaking changes.

Since we didn’t receive a CI status on the [`greenkeeper/initial`](https://github.com/fastify/create-fastify/commits/greenkeeper/initial) branch, it’s possible that you don’t have CI set up yet.
We recommend using:
- [CircleCI](https://circleci.com)
- [Travis CI](https://travis-ci.org)
- [Buildkite](https://buildkite.com/)
- [CodeShip](https://codeship.com)
- [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines)
- [TeamCity](https://www.jetbrains.com/teamcity)
- [Buddy](https://buddy.works)
- [AppVeyor](https://www.appveyor.com)
But Greenkeeper will work with every other CI service as well.

If you _have_ already set up a CI for this repository, you might need to check how it’s configured. Make sure it is set to run on all new branches. If you don’t want it to run on absolutely every branch, you can whitelist branches starting with `greenkeeper/`.

Once you have installed and configured CI on this repository correctly, you’ll need to re-trigger Greenkeeper’s initial pull request. To do this, please click the 'fix repo' button on [account.greenkeeper.io](https://account.greenkeeper.io).

### PR DESCRIPTION
**Pull request by veritem**
Fix #1 #2

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
Can you remove `package-lock.json` and add it to `.gitignore`.. also remove `.travis.yml`.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (mcollina) [inline .github/workflows/ci.yml]: Can you set this up like the rest of the repos in this org?

**Review by mcollina — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (veritem) [reply .github/workflows/ci.yml]: I updated

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
```
Comment (mcollina) [inline .github/workflows/ci.yml:3]: This is what I mean: https://github.com/fastify/fastify/blob/7db09f107d83e6f85907e43199ac945e4d761e52/.github/workflows/ci.yml#L3

**Review by mcollina — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**